src/components/ChatLog.js

```diff
@@ -0,0 +1,39 @@
+import ChatEntry from '../components/ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+
+  const fullChatLog = props.entries.map((entry, index) => {
+    return (
+      <li key={index}>
+        <ChatEntry
+          id={entry.id}
+          sender={entry.sender}
+          body={entry.body}
+          timeStamp={entry.timeStamp}
+          liked={entry.liked}
+          onHandleLikes={props.onUpdateLikes}
+        ></ChatEntry>
+      </li> 
+      )
+    });
+
+  return (
+    <div className="fullChatLog">
+      {fullChatLog}
```

Comment:
While Learn does show using a variable to reference the return value from calling .map() and then using the variable in the JSX like you've done here, I did want to call out that you'll see another, more common way of writing code that iterates over a list of objects:
```javascript
return (
    <ul>
        {props.entries.map.((entry, index) => {
            <li key={index}>
            <ChatEntry
                id={entry.id}
                sender={entry.sender}
                body={entry.body}
                timeStamp={entry.timeStamp}
                liked={entry.liked}
                onHandleLikes={props.onUpdateLikes}>
            </ChatEntry>
            </li> 
            }
        }
)